features/FxA-83-sign-in-confirmation-email/README.md

```diff
@@ -222,28 +222,57 @@ against unverified tokens.
 
 ## Results
 
-Sign-in confirmation was deployed over several train cycles.
-The feature was turned on at 6-24-2016 for 2% of our users and
-gradually increased to 100%. During the deployment we monitored
-the sign-in success and error rates to ensure that users were
-not negatively impacted.
+Sign-in confirmation was deployed over several
+train cycles. The feature was turned on at 6-24-2016
+for 2% of our users and gradually increased to 100%.
+During the deployment we monitored the sign-in success
+and error rates to ensure that users were not negatively
+impacted.
 
 To help gauge the success of the feature, several dashboards
 were created.
 
 ### Sync sign-in success rate
 
-Pre sign-in confirmation metrics show that we had between a 42% to 52%
-sync sign-in success rate. From [first dashboard](https://app.datadoghq.com/dash/163668/fxa-content---signin-confirmation), with sign-in confirmation, we have maintained this sign-in rate.
+Pre sign-in confirmation metrics show that we had
+between a 42% to 52% sync sign-in success rate.
```

Comment:
I have trouble with this metric when I look at the details in datadog.
(fxa.content.signin.signin.success + fxa.content.force_auth.signin.success + fxa.content.complete_signin.verification.success) / (fxa.content.signin.submit + fxa.content.force_auth.submit) * 100
Why are there 3 different login success events?
Why are there 2 different form submit events?

I've tried to piece things together ([by reading this](https://github.com/mozilla/fxa-content-server/blob/master/docs/client-metrics.md)) but it's a bit unclear to me and some events are missing I think.